### ISSUE TITLE
Blueprint proposal discussion: EP011

### ISSUE THREAD

**Issue by renanrodrigo** [kytos/kytos#311]
Propose an easier way to create Multipart Request messages using python-openflow. [EP011](https://github.com/kytos/kytos/blob/master/docs/blueprints/EP011.rst)

**Comment by erickvermot** [comment]
I agree the current process to create complex messages (not only MP ones), involves too many steps and require some knowledge of the specs (which I think we could avoid). I believe this EP also relates to [EP009](https://github.com/kytos/kytos/blob/master/docs/blueprints/EP009.rst), that intends to propose an easier way to create messages.

About the proposed solution, I would just change ``SpecificRequestBody()`` to ``SpecificRequest()``.

**Comment by beraldoleal** [comment]
@erickvermot I know that EP009 IF APPROVED can be related with this. But for now, lets treat them independently. EP009 is much more radical.

**Comment by erickvermot** [comment]
ok @beraldoleal. I do think this EP011 is a good step.
Maybe similar dev could be made for other messages like the ``FlowStatsRequest()``,
what do you guys think?
